src/oiiotool/oiiotool.cpp

```diff
@@ -1792,9 +1828,15 @@ set_full_to_pixels(int argc, const char* argv[])
     Timer timer(ot.enable_function_timing);
     string_view command = ot.express(argv[0]);
 
+    std::map<std::string, std::string> options;
```

Comment:
Any chance these repeated 4 lines could be put into a function somewhere?

Reply:
Well, I'm planning an overhaul of this soon anyway, and it's going to change a lot, including a lot less repetition of some of these idioms.